docs/docs/adding-forms.md

```diff
@@ -173,6 +173,6 @@ export default class IndexPage extends React.Component {
 }
 ```
 
-This form isn't doing anything besides showing the user information that they just entered. At this point, you may want to move this form to a component, send the form state to a backend server, or add robust validation. You can also use fantastic React form libraries like [Formik](https://github.com/jaredpalmer/formik) or [Final Form](https://github.com/final-form/react-final-form) to speed up your development process.
+हा फॉर्म वापरकर्त्याची माहिती दाखवण्याव्यतिरिक्त काही अतिरिक्त करत नाही. या वेळी, आपण हा फॉर्म एका कॉम्पोनन्ट कडे हलवू शकता, फॉर्म स्टेट बॅकएंड सर्वरवर पाठवा, किंवा रोबस्त मान्यता जोडा. तुम्ही अप्रतिम रिऍक्ट देखील वापरू शकता, जसे लायब्ररी [Formik](https://github.com/jaredpalmer/formik) किव्हा [Final Form](https://github.com/final-form/react-final-form) आपल्या विकास प्रक्रियेला गती देण्यासाठी वापरू शकता.
```

Comment:
replace with this किंवा

Reply:
Definitely, I will do the needful.
Thanks!